### PR TITLE
Add Japanese translations

### PR DESCRIPTION
**Pull request by okaits**
Added Japanese translations.﻿

### CODE REVIEW

File: src/i18n/rpi-imager_ja.ts [hunk 1]
```diff
@@ -0,0 +1,752 @@
...
+        <location filename="../main.qml" line="524"/>
+        <location filename="../main.qml" line="1025"/>
+        <source>削除</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location filename="../main.qml" line="525"/>
+        <source>SDカードをFAT32としてフォーマットする</source>
+        <translation type="unfinished"></translation>
+    </message>
```
Comment (lurch) [inline src/i18n/rpi-imager_ja.ts:598]: What's happened here? Are the `source` and `translation` accidentally the wrong way around?

**Comment by lurch** [comment]
I can't read Japanese, but it looks like some of the translations might be missing the `%1` placeholder that is present in the source text? :shrug:

File: src/i18n/rpi-imager_ja.ts (hunk 1 above)
Comment (okaits) [reply src/i18n/rpi-imager_ja.ts]: Oh, I made a mistake.
Fixed at 6a0bd8b.

**Comment by okaits** [comment]
Oh, I'm sorry. It seems that my environment was broken. I'll revert d8edef0.

**Comment by okaits** [comment]
I'm sorry, I'm getting confused, I'll create the PR again.

**Pull request closed without merging**